src/api/spec/factories/project.rb

```diff
@@ -11,18 +11,13 @@
 
     factory :project_with_package do
       transient do
-        package_name nil
+        sequence(:package_name) { |n| "package_#{n}" }
         create_patchinfo false
         maintainer nil
       end
 
       after(:create) do |project, evaluator|
-        new_package = if evaluator.package_name
-                        create(:package, project_id: project.id, name: evaluator.package_name)
-                      else
-                        create(:package, project_id: project.id)
-                      end
-        project.packages << new_package
+        project.packages << create(:package, project_id: project.id, name: evaluator.package_name)
```

Comment:
ah, that got changed .... maybe Project should handle the name as primary key then as well ... but that will break with a project named "123" :/